main.go

```diff
@@ -151,6 +132,41 @@ func main() {
 	}
 }
 
+func addNewWords(target string, nowStreak *int, newWords *int, anki *Anki, args map[string]interface{}) (error) {
```

Comment:
Could you please avoid using args here? Maybe just pass `format string` and `dec string` as two arguments.